src/app/location/location.component.ts

```diff
@@ -83,6 +84,12 @@ export class LocationComponent implements OnInit, OnDestroy {
 
           if (data.length) {
             this.loadChartData(0);
+            let lastElement = this.mainData[this.mainData.length - 1];
```

Comment:
/n after variable declaration.

Also, are we somehow mutating lastElement, could this be a `const`?

Reply:
At first it was const, but VS Code gave it strange color but worked ok, so changed to let.
But I'll return it to const :)